plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp

```diff
@@ -102,6 +102,7 @@ struct txn_test_gen_plugin_impl {
    name                                                 newaccountA;
    name                                                 newaccountB;
    name                                                 newaccountT;
+   fc::microseconds                                     trx_expiration;
```

Comment:
Shouldn't this be `fc::seconds`?

Reply:
fc::seconds() is a function which return fc::microseconds object.